closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -201,6 +256,17 @@ static void printErrors(void) {
     fflush(stderr);
 }
 
+void OSSL_RSA_get0_key_compatibility(const RSA *r,
+                  const BIGNUM **n, const BIGNUM **e, const BIGNUM **d)
+ {
+    if (n != NULL)
+        *n = r->n;
+    if (e != NULL)
+        *e = r->e;
+    if (d != NULL)
+        *d = r->d;
+ }
```

Comment:
Opening and closing braces (`{}`) should be at the beginning of those lines.
Bodies of `if` statements should be enclosed in braces.

Reply:
You mean like [this](https://github.com/ibmruntimes/openj9-openjdk-jdk11/blob/b39d79d05e1ad38d6c032589d17c9ad992a0585f/closed/src/java.base/share/native/libjncrypto/NativeCrypto.c#L263-L271), right?

Reply:
Better, but there's still an extra space before the braces on lines 261 and 268.